tests/app/clients/test_aws_ses.py

```diff
@@ -190,7 +190,7 @@ def test_send_email_does_not_raise_AwsSesClientThrottlingSendRateException_if_no
         )
 
 
-def test_send_email_raises_other_errs_as_AwsSesClientException(mocker):
```

Comment:
`mocker` should generally come after `notify_api` so that the teardowns happen in the right order.